### ISSUE TITLE
Alert when connection is lost

### ISSUE THREAD

**Issue by Zahnweh** [lucavb/homebridge-esphome-ts#88]
Hello, it would be good to know when the connection has been lost. Up to now, the last transmitted value is still displayed after a connection break. It then looks as if the sensor is still online, although it has gone offline. Testet with a DS18b20 sensor. Thanks.

**Comment by lucavb** [comment]
Hey, the documentation seems a bit unclear on how to realise this with Homebridge but it would appear that the way to go is the one I implemented here. So please download this file to your Homebridge server and run 
`npm i -g homebridge-esphome-ts-1.2.1.tar.gz`

[homebridge-esphome-ts-1.2.1.tar.gz](https://github.com/lucavb/homebridge-esphome-ts/files/6259153/homebridge-esphome-ts-1.2.1.tar.gz)

**Comment by Zahnweh** [comment]
Hey, thanks a lot for your rapid action on this. I downloaded and installed as you suggested. It told me I did not have permissions so I used sudo which finally worked out. But when I reboot homebridge it says "Warning: skipping plugin found at '/usr/lib/node_modules/homebridge-esphome-ts' since we already loaded the same plugin from '/usr/local/lib/node_modules/homebridge-esphome-ts'." As the behavior of the plugin is still the same, I think the one that is being skipped is the one you send me. I'd be glad being able to correct this so that I can test it for you.

**Comment by lucavb** [comment]
Interesting. So maybe run `npm r -g homebridge-esphome-ts` to get rid of the previous version of yours and then rerun the command. That should hopefully do the trick.

**Comment by Zahnweh** [comment]
Hi, thanks again! I uninstalled with `sudo npm r -g` and reinstalled with sudo `npm i -g` but it went back into /usr/lib/node_modules/ again. I have then deleted the existing instance of the plugin in /usr/local/lib/node_modules/ and moved the other one to this folder with `sudo mv`. After all, your new build should now be the one that is installed and working. However, I cannot determine any changes when the sensor is disconnected. How should the changes you made be recognized?

**Comment by lucavb** [comment]
Can you see from the logs that the plugin detected the espdevice as offline? It should print something like 
`The device under the host <Hostname> could not be reached.`

**Comment by Zahnweh** [comment]
Yes, this is what it does: [10/04/2021, 11:55:12] [esphome] The device under the host 192.168.0.105 could not be reached. But  it's still displaying the last value that has been transmitted before it went offline.
